.github/workflows/unit-test-php.yml

```
@@ -0,0 +1,119 @@
+name: PHPunit with database
+
+on:
+  workflow_call:
+    inputs:
+      versions:
+        default: >-
+          ["8.1"]
+        description: "PHP versions"
+        required: false
+        type: string
+      php_extensions:
+        default: dom, curl, libxml, mbstring, mailparse, zip, pcntl, pdo, pdo_mysql
+        description: "PHP extensions to enable"
+        required: false
+        type: string
+      fail_on_low_coverage:
+        default: false
+        description: "Should this workflow fail when the coverage is below the minimum"
+        required: false
+        type: boolean
+      coverage_low:
+        default: 60
+        description: "Minimal required percentage of code that should be covered by tests, optionally fail when coverage is below minimum"
+        required: false
+        type: number
+      coverage_high:
+        default: 80
+        description: "Minimal optimal percentage of code that should be covered by tests"
+        required: false
+        type: number
+      pre_tests_script:
+        default:
+        description: "A bash script to run before running php unit"
+        required: false
+        type: string
+      env_file:
+        default: .env.testing
+        description: "Which env file to use as default"
+        required: false
+        type: string
+
+jobs:
```

Add forced secrets for our images

```suggestion
    secrets:
      dockerhub-username:
        description: Docker Hub username
        required: true
      dockerhub-token:
        description: Docker Hub token
        required: true

jobs:
```